presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.hive;
+
+import io.prestosql.tempto.Requirement;
+import io.prestosql.tempto.RequirementsProvider;
+import io.prestosql.tempto.configuration.Configuration;
+import org.testng.annotations.Test;
+
+import java.math.BigDecimal;
+
+import static io.prestosql.tempto.Requirements.compose;
+import static io.prestosql.tempto.assertions.QueryAssert.Row.row;
+import static io.prestosql.tempto.assertions.QueryAssert.assertThat;
+import static io.prestosql.tempto.fulfillment.table.TableRequirements.immutableTable;
+import static io.prestosql.tempto.fulfillment.table.hive.tpch.TpchTableDefinitions.NATION;
+import static io.prestosql.tempto.query.QueryExecutor.query;
+import static io.prestosql.tests.TestGroups.HIVE_VIEWS;
+import static io.prestosql.tests.utils.QueryExecutors.onHive;
+import static io.prestosql.tests.utils.QueryExecutors.onPresto;
+
+public class TestHiveViews
+        extends HiveProductTest
+        implements RequirementsProvider
+{
+    @Override
+    public Requirement getRequirements(Configuration configuration)
+    {
+        return compose(
+                immutableTable(NATION));
+    }
+
+    @Test(groups = HIVE_VIEWS)
+    public void testSelectOnView()
+    {
+        onHive().executeQuery("DROP VIEW IF EXISTS hive_test_view");
+
+        onHive().executeQuery("CREATE VIEW hive_test_view AS SELECT * FROM nation");
+
+        assertThat(query("SELECT * FROM hive_test_view")).hasRowsCount(25);
+    }
+
+    @Test(groups = HIVE_VIEWS)
+    public void testSelectOnViewFromDifferentSchema()
```

Comment:
This seems to belong to the next commit: `Support Hive views with tables from unspecified and different schema`.

Squash?

Reply:
squashed